internal/runtime/compiler/parser/parser.y

```diff
@@ -73,6 +77,7 @@ import (
 %token LCURLY RCURLY LPAREN RPAREN LSQUARE RSQUARE
 %token COMMA
 %token NL
+%token LOGMAPPING
```

Comment:
I think I don't like the keyword name `logmapping`.

Can you try calling it `log_filter`, `logname_filter`, or `filename_filter` or something like that?  I think the correct verb for a user is to filter the log filename, not to map a log to a program, despite the bug's history.

I think `log_filter` is the most descriptive as well as not too verbose.

Reply:
Changed to match suggestion